network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/pixels/NetworkProtectionPixelNames.kt

```diff
@@ -47,6 +47,8 @@ enum class NetworkProtectionPixelNames(
     NETP_WG_ERROR_FAILED_TO_LOAD_WG_LIBRARY("m_netp_ev_wireguard_error_unable_to_load_wireguard_library_c", enqueue = true),
     NETP_WG_ERROR_FAILED_TO_LOAD_WG_LIBRARY_DAILY("m_netp_ev_wireguard_error_unable_to_load_wireguard_library_d", enqueue = true),
     NETP_LATENCY_REPORT("m_netp_ev_latency_c"),
+    NETP_REPORT_POOR_LATENCY_DAILY("m_netp_ev_poor_latency_d"),
```

Comment:
We should probably add this in https://app.asana.com/0/1203137811378537/1203882129166495/f